eva-accession-pipeline/src/main/java/uk/ac/ebi/eva/accession/pipeline/io/AccessionWriter.java

```diff
@@ -15,10 +15,24 @@
  */
 package uk.ac.ebi.eva.accession.pipeline.io;
 
-import org.springframework.batch.item.support.CompositeItemWriter;
+import org.springframework.batch.item.ItemWriter;
 
-import uk.ac.ebi.eva.accession.core.persistence.SubmittedVariantEntity;
+import uk.ac.ebi.eva.accession.core.ISubmittedVariant;
+import uk.ac.ebi.eva.accession.core.SubmittedVariantAccessioningService;
 
-public class SummaryFileWriter extends CompositeItemWriter<SubmittedVariantEntity> {
-    // TODO: implement
+import java.util.List;
+import java.util.Map;
+
+public class AccessionWriter implements ItemWriter<ISubmittedVariant> {
```

Comment:
If this class consumes `ISubmittedVariant` objects, does that mean we can change the processor to have the interface as output type?

Reply:
yes, we can. it doesn't matter.
`public SimpleStepBuilder<I, O> processor(org.springframework.batch.item.ItemProcessor<? super I, ? extends O> processor)`